README.md

```diff
@@ -68,10 +68,10 @@ socket.connect(socketPort)
 Generating a native binary:
 ---------------------------------
 To generate a native binary:
-- [Install GraalVM 20.2.0](https://www.graalvm.org/docs/getting-started/#install-graalvm)
-- In a terminal, run `gu install native-image`
-- Execute a Maven build that sets the flag `native`: `./mvnw clean package -Dnative -DskipTests`
-  - On Linux, compile with `./mvnw clean package -Dnative -DskipTests -Dgraalvm.static=--static`
+- [Install GraalVM](https://www.graalvm.org/docs/getting-started/#install-graalvm),
+  ideally with a tool that allows setting up multiple versions of Java, such as SDKMAN
+- Execute a Maven build that sets the profile `native`: `./mvnw clean package -Pnative -DskipTests`
+  - On Linux, compile with `./mvnw clean package -Pnative -DskipTests -Dgraalvm.static=--static`
```

Comment:
Can we continue to activate it with `-D` as opposed to `-P`. I have always been nervous about activating profiles explicitly like this because IIRC they override (disable) any profiles that would be activated by default due to some condition. As a result I've always made it possible to activate profile by properties.

Also, just something to be aware, just giving `--static` to that property is technically the `alpine` configuration : https://github.com/redhat-developer/vscode-xml/blob/f66692877045eb72460987523d6b03686a4af9fb/.github/workflows/native-image.yaml#L35-L40 . The Linux one has an extra flag to avoid statically linking libc, but I think either should work, and given that the current is a smaller command to type (and this is just for trying it out), maybe we leave it as-is.